newsletter-archive/index.html

```diff
@@ -181,7 +181,20 @@ <h4 class="title">2024</h4>
                                            data-track-title="August 2024 Developer Newsletter"
                                            data-track-text="August 2024 Developer Newsletter"
                                            title="August 2024 Developer Newsletter">
-                                           → August (latest)
+                                           → August 
```

Comment:
remove `→` from August. Only latest archive should have it.

Reply:
Sorry, I missed it. Now it is done. Tnx